platform/mime/core/platform-mime-core-impl/pom.xml

```diff
@@ -75,6 +77,15 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.tika</groupId>
+            <artifactId>tika-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.codice.thirdparty</groupId>
+            <artifactId>tika-bundle</artifactId>
```

Comment:
You generally don't use a bundle as a dependency, you use a jar. The one that you actually need is most likely tika-parsers. The bundles only belong in the features files.